src/NzbDrone.Core/ImportLists/FetchAndParseImportListService.cs

```diff
@@ -63,7 +68,7 @@ public List<ImportListItemInfo> Fetch()
 
                              lock (result)
                              {
-                                 _logger.Debug("Found {0} from {1}", importListReports.Count, importList.Name);
+                                 _logger.Debug("Found {0} from [{1}]: {2}", importListReports.Count, importList.Name, importListLocal.Definition.Name);
```

Comment:
`Found {0} from {1} ({2})` reads a lot better, instead of the definition name reading like a sub title. Same thing for the other places you added this formatting please.